### DISCUSSION TITLE
Can I somehow query the ctags file from the command line?

### DISCUSSION THREAD

**Discussion by madorian** [universal-ctags/ctags#4049] [General]
I'm not really understanding ctags. I use it with C# and I get weird results, so I need to just get the basics tested. 

If I generate a ctags file from some simple files, can I query using some cli tools?

**Comment by masatake** [comment]
`tags` file is not so complicated. So, if it is small, just reading the file is the simple way for debugging. https://docs.ctags.io/en/latest/man/tags.5.html explains the format.

`readtags` is a command for querying. It is part of this (universal-ctags/ctags) repository.
See https://docs.ctags.io/en/latest/man/readtags.1.html
